src/app/api/payment-address/route.ts

```diff
@@ -0,0 +1,21 @@
+import { getServerSession } from "next-auth";
+import { NextRequest, NextResponse } from "next/server";
+import { options } from "../auth/[...nextauth]/options";
+import { fetchTeamPaymentAddresses } from "./service/paymentAddress";
+
+export async function GET(req: NextRequest) {
```

Comment:
As you used payments on the other one (/contributors/payments)
I think we should move this to /payments only as well, otherwise gets pretty confusing.

No need to do it now tho